snort-ip-blocklist-feed/connector.py

```diff
@@ -0,0 +1,27 @@
+""" Copyright start
```

Comment:
Please update the copyright with the below lines.

"""
Copyright start
MIT License
Copyright (c) 2023 Fortinet Inc
Copyright end
"""